docs/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
Is this necessary because of sphinx ? It looks like a Mac-specific problem, we might want to keep that out of the `.gitignore` and individually have it in `~/.git/info/exclude` or equivalent.

Reply:
I don't think so; guess Kay added it because she's on a Mac. I'll remove it.

Reply:
538229cf6fbbe2a5ab9b5504b290c78beef851d1